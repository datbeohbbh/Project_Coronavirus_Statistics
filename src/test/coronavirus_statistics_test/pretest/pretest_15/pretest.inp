26 22
? 5
? 5
q 1 25 20
q 1 17 186
q 1 3 218
? 4 2
q 1 26 363
q 2 26 320
? 4 1
? 6
q 1 14 204
q 3 14 9
? 4 3
? 7
? 7
? 7
? 6
? 6
? 4 1
? 4 3
? 6
? 4 2