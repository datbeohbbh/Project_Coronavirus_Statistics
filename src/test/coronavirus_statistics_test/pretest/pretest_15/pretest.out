Query ? 5 :
The number of safe cities is 26

Query ? 5 :
The number of safe cities is 26

Query ? 4 2 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 186
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 20
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 320 people

Query ? 4 3 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 186
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 14
Quantity: 9 people

Query ? 7 :
The city with the highest death is city with the index 14
Quantity: 9 people

Query ? 7 :
The city with the highest death is city with the index 14
Quantity: 9 people

Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 320 people

Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 320 people

Query ? 4 1 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 20
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 186
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 26
Quantity: 320 people

Query ? 4 2 :
The answer is the city with the index 26 
- Information of the city 26 -
-> Number of current patients : 43
-> Has recovered : 320
-> Was dead : 0

